### PR TITLE
pm, settings: refresh pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Optional and mandatory fixes from https://github.com/tldr-pages/tldr/pull/9696 were applied to the original pages.

### CODE REVIEW

File: pages/android/pm.md [hunk 1]
```diff
@@ -3,22 +3,22 @@
 > Show information about apps on an Android device.
```
Comment (kbdharun) [inline pages/android/pm.md:3]:
```suggestion
> Display information about apps on an Android device.
```

File: pages/android/pm.md [hunk 2]
```diff
@@ -3,22 +3,22 @@
...
 > More information: <https://developer.android.com/studio/command-line/adb#pm>.
 
-- Print a list of all installed apps:
+- List of all installed apps:
```
Comment (kbdharun) [inline pages/android/pm.md:6]:
```suggestion
- List all installed apps:
```

File: pages/android/pm.md [hunk 3]
```diff
@@ -3,22 +3,22 @@
...
 
 `pm list packages`
 
-- Print a list of all installed system apps:
+- List of all installed system apps:
```
Comment (kbdharun) [inline pages/android/pm.md:10]:
```suggestion
- List all installed system apps:
```

File: pages/android/pm.md [hunk 4]
```diff
@@ -3,22 +3,22 @@
...
 
 `pm list packages -s`
 
-- Print a list of all installed 3rd-Party apps:
+- List of all installed 3rd-Party apps:
```
Comment (kbdharun) [inline pages/android/pm.md:14]:
```suggestion
- List all installed 3rd-Party apps:
```

File: pages/android/pm.md [hunk 5]
```diff
@@ -3,22 +3,22 @@
...
 
 `pm list packages -3`
 
-- Print a list of apps matching specific keywords:
+- List of apps matching specific keywords:
```
Comment (kbdharun) [inline pages/android/pm.md:18]:
```suggestion
- List apps matching specific keywords:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**